packages/common/src/hooks/useImageSize.ts

```diff
@@ -46,7 +46,7 @@ export const useImageSize = <
 
       while (mirrors.length > 0) {
         try {
-          await preloadImageFn(currentUrl)
+          await preloadImageFn?.(currentUrl)
```

Comment:
What's the effect if a preload fn isn't passed / isn't called? It just gets loaded the normal way once the browser renders it?

Reply:
yep! that's right, i can add a comment